PowerShell/JumpCloud Module/JumpCloud.psm1

```diff
@@ -52,52 +52,22 @@ If ($PSVersionTable.PSEdition -eq 'Core') {
 # Set function aliases
 Set-Alias -Name:('New-JCAssociation') -Value:('Add-JCAssociation')
 # Argument completers
-# Populate values for function parameters. "Dynamic ValidateSet"
+# set Argument Completer(s) which no not require authentication
```

Comment:
Just a note here, the filters in our documentation were not always correct, I've taken the argument completer out of the equation here. An added benefit is that the module loads much faster now. 

TODO: Team to check that `Get-JCSystemInsight -Table *tab*` still shows valid tables